index.html

```diff
@@ -319,6 +321,18 @@ <h2>Glossary</h2>
 
 
 
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="j">J</div>
+
+<p><dfn class="lint-ignore export">Jamo</dfn>. The basic unit used to form Hangul syllables, representing vowels and consonants in Korean. Besides the code points for Jamos, Unicode has 11,172 Hangul <em>syllables</em> that represent jamo combinations encoded as single characters. In practice, they are the main characters in actual use. (See also: <a href="https://www.unicode.org/faq/korean.html">Unicode Korean FAQ</a>)</p>
+
+<a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
+
+<div class="letter_anchor" id="k">K</div>
+
+<p><dfn class="lint-ignore export">Kana</dfn>. <a href="https://unicode.org/glossary/#kana">Unicode definition</a>: <q cite="https://unicode.org/glossary/#kana">The name of a primarily syllabic script used by the Japanese writing system. It comes in two forms, <a href="https://unicode.org/glossary/#hiragana">hiragana</a> and <a href="https://unicode.org/glossary/#katakana">katakana</a>. The former is used to write particles, grammatical affixes, and words that have no <a href="https://unicode.org/glossary/#kanji">kanji</a> form; the latter is used primarily to write foreign words.</q></p>
```

Comment:
Sorry, just one last comment.  The text 'a primarily syllabic script' is not really correct.  It's a fully syllabic script, i think.  Note that the n is syllabic in Japanese, where the syllables are based on mora.  So i'd suggest removing the 'primarily' word.

Reply:
@r12a I tend to think of _hiragana_ and _katakana_ as two distinct scripts, both of which encode the Japanese syllabary and thus `kana` is a collective noun that refers to both?

On the other hand, @xfq is directly and exactly quoting the Unicode glossary here. Should we file bugs against "primarily" and or suggest changes to their glossary first?

Reply:
Yes.

Reply:
Fixed by quoting the latest Unicode glossary. Thanks.